src/dc_tools.rs

```diff
@@ -1555,6 +1557,69 @@ pub fn time() -> i64 {
         .as_secs() as i64
 }
 
+/// Very simple email address wrapper.
+///
+/// Represents an email address, right now just the `name@domain` portion.
```

Comment:
is this enough to be useful?  Are incoming messages's To-header parsed by this code or is that happening inside mmime?

Reply:
this is more than the c code did before (it just checked for a single `@` symbol in one path)

Reply:
ah ok!